templates/tfengine/recipes/org/org_policies.hcl

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 template "terragrunt" {
-  component_path = "../../components/terragrunt/leaf"
+  recipe_path = "../terraform/terragrunt.hcl"
```

Comment:
why does it become a recipe?

Reply:
Terragrunt recipe no longer sets up the terraform configs, only terragrunt.hcl